### PR TITLE
Hotfix the issues

### PR DESCRIPTION
**Pull request by sureshHARDIYA**
Hi Guys, 

I reviewed the code, fixed it. now, it is working. Tested with `Observation` with the `mutation` feature. Rest, you guys should test it. 

# NOTE Following things:

image

You can NOT refer to a Model that does not exist. 
```
appointment: [{
	type: 'ObjectID',
	ref: 'Appointment',
 }],
```
Here, I assume there is a mongo Model with name `Appointment` which I could not locate. 

Screen Shot 2020-02-15 at 16 24 48

You can not create virtual attributes and assign something that does not exist.

Read more here: https://mongoosejs.com/docs/guide.html#virtuals

Basically, before linking anything, it needs to be properly linked.

### CODE REVIEW

**Comment by martinheitmann** [comment]
Thanks a lot for fixing the issue! I've tested mutation create and query with both Media and Observation, and it works as expected.
